src/components/panel/panelSections/panelContent.component.tsx

```diff
@@ -105,76 +112,96 @@ export const PanelContent = ({
     onReorder(updatedObjects);
   };
 
+  if (draggedObject)
+    return (
+      <div
+        ref={setNodeRef}
+        className={clsx(
+          isOver && "border-primary text-primary",
+          "m-4 mt-10 flex h-72 items-center justify-center border-2 border-dotted text-center text-manatee-400",
+        )}
+      >
+        <span>{"Drop object here to add to the Set's content"}</span>
+      </div>
+    );
+
   return (
-    <Reorder.Group
-      axis="y"
-      values={objects}
-      onReorder={onReorder}
-      className="h-full overflow-y-auto p-4 pt-6 text-sm md:p-8"
-      data-testid="panel-content-items"
-    >
-      {objects.map((item, index) => {
-        const { object, config, position } = item;
+    <>
+      {inEditMode && (
+        <span className="mt-16 text-center text-sm text-manatee-600">
```

Comment:
Reduce the padding/margin above and below to reduce the layout movement when you switch to edit